Ingress/controllers/nginx-third-party/main.go

```diff
@@ -62,14 +62,24 @@ func main() {
 		glog.Fatalf("Please specify --default-backend")
 	}
 
+	glog.Info("Checking if DNS is working")
+	ip, err := checkDNS(*defaultSvc)
```

Comment:
> can we run this without dns?

Yes but then we need to specify the ip of the service. Right now it just try to resolve `svcName.namespace` to an [IP address](https://github.com/kubernetes/contrib/pull/512/files#diff-e6b8f3022373cce5f55cdfce8ad4516cR234) (maybe I need to remove the word add-on, it will work with any dns capable to resolve name->ip)

Just in case dns is a requirement [check here](https://github.com/kubernetes/contrib/blob/master/Ingress/controllers/nginx-third-party/nginx.tmpl#L237) 
I'm just trying to avoid to start nginx and not be able to serve content because